steps/bitrise-snyk-scan/0.1.0/step.yml

```diff
@@ -0,0 +1,54 @@
+title: bitrise-snyk-scan
```

Comment:
The title is usually a human-readable string in Title Case. I suggest replacing this with something like `Bitrise Snyk Scan`.

Reply:
Sure, I'll change it.

Reply:
Done